pkg/katautils/config.go

```diff
@@ -368,7 +389,7 @@ func (n netmon) debug() bool {
 	return n.Debug
 }
 
-func newQemuHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
+func newFirecrackerHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
```

Comment:
Nit: Adding the firecracker function above the existing one make reviewing this change harder ;(

Reply:
Ah, I did it alphabetical order.  noted.